sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Improving the Kubernetes UX for cluster administration
+- Kubernetes cluster lifecycle related features and issues
+- Tools that assist in bootstrapping, mutating, and upgrading Kubernetes control plane components. 
+- Kubernetes tools used to manage infrastructure related to provisioning and upgrading nodes
+- Tools that assist in management of configuration of Kubernetes components
+
+#### Code, Binaries and Services
+
+- Everything that falls in the scope of the SIG.
+- Tools that are provider specific implementation for infrastructure management.
+- Core add-ons that are required for cluster bootstrapping.
+
+#### Cross-cutting and Externally Facing Processes
+
+- This SIG works closely with SIG Release during the end of the cycle, because of how the release is currently managed. The process requires a lock-step coordination which is outlined [here](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md)
+- The SIG recommends and verifies compatibility of critical cluster add-ons for networking, network policy, service discovery, etc. The SIG maintains the health check of container images for some add-ons that are required for cluster bootstrapping. While the SIG could provide support to users that have add-on related issues, the SIG can decide to delegate issues to the add-on maintainers or other SIGs.
+- The SIG owns the framework for upgrade / downgrade testing of Kubernetes. However, the SIG does not own the individual tests for each feature. While the SIG is often involved in triaging and debugging upgrade or downgrade test failures, the SIG will normally delegate issues to the feature owner for the failing test.
+- The SIG collaborates regularly with SIG Auth in an effort to follow best practices in order to promote secure default deployments.
+- The SIG co-owns cloud provider specific code related to cluster and machine provisioning with the respective SIGs for each cloud provider but does not own the cloud controller manager or any other provider specific code
+
+### Out of scope
+
+- Networking related issues (see [sig-network](../sig-network))
+- UI / UX issues other than cluster bootstrapping or management (see [sig-ui](../sig-ui) and  [sig-cli](../sig-cli))
+- Kubernetes control plane issues:
+   - Node related issues (see [sig-node](../sig-node))
+   - Control plane component related issues (see [sig-apimachinery](../sig-apimachinery))
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Chairs
+
+- Selecting feature for a Kubernetes milestone.
+- Host the weekly SIG meeting, ensure that recordings are uploaded in a timely fashion
```

Comment:
nit: (this is a super nit, but it bugs me when subsequent bullets use different tense) s/Host/Hosting